client/js/lounge.js

```diff
@@ -37,6 +37,22 @@ $(function() {
 		"/voice",
 		"/whois"
 	];
+	var handledTypes = [
```

Comment:
Mostly performance. Looking up a key is a lot faster than iterating the entire array to find it.